pkg/build/builder/docker.go

```diff
@@ -149,10 +152,6 @@ func (d *DockerBuilder) Build() error {
 		return err
 	}
 
-	cname := containerName("docker", d.build.Name, d.build.Namespace, "post-commit")
-
-	err = execPostCommitHook(ctx, d.dockerClient, d.build.Spec.PostCommit, buildTag, cname)
-
 	if err != nil {
```

Comment:
this block is dead code, it will never be true.

Reply:
removed in next push